docs/topics/testing/tools.txt

```diff
@@ -1566,6 +1566,38 @@ your test suite.
 
     Output in case of error can be customized with the ``msg`` argument.
 
+.. method:: SimpleTestCase.assertSignalSent(signal, required_kwargs=None)
+
+    Asserts that a signal was sent. This assertion **must** be used as
+    a context manager::
+
+        with self.assertSignalSent(signals.post_save):
+            m = MyModelInstance()
+            m.save()
+
+    If ``required_kwargs`` is provided, also asserts that the signal
+    was sent with keyword arguments of those names::
+
+        with self.assertSignalSent(my_signal, required_kwargs=['my_kwarg']):
+            trigger_signal_send()
+
+    To access detailed information about the arguments sent with the
+    signal, inspect the ``received_kwargs`` and ``sender`` attributes
+    of the context manager::
+
+        with self.assertSignalSent(my_signal) as cm:
```

Comment:
This code example is not realistic without some line of code inside the context manager that might trigger the signal.